build.sbt

```diff
@@ -232,11 +209,10 @@ lazy val noPublishSettings = Seq(
 lazy val toolSettings =
   baseSettings ++
     Seq(
-      crossSbtVersions := List(sbt13Version, sbt10Version),
+      crossSbtVersions := List(sbt10Version),
```

Comment:
You might want to set this to an older version of sbt unless you are using sbt 1.3.x features. Otherwise you might publish a version of the plugin that works on sbt 1.3.x but not earlier versions. See https://github.com/sbt/sbt/issues/5154.

Reply:
Thanks for the review. What version of sbt `1.x` do you recommend?

Reply:
`1.2.8` according to https://github.com/sbt/sbt/issues/5154#issuecomment-538666595